lib/ansible/runner.rb

```diff
@@ -231,6 +231,12 @@ def run_via_cli(hosts, credentials, env_vars, extra_vars, tags: nil, ansible_run
                    end
 
           res = response(base_dir, ansible_runner_method, result, debug)
+          if !cred_env_vars.empty?
+            command_line = res.instance_variable_get(:@command_line)
+            new_command = "export #{cred_env_vars['ANSIBLE_VAULT_PASSWORD_FILE']} ; #{command_line}"
```

Comment:
This is interesting.  I feel like there's got to be a better way to do this, but at the moment I'm willing to merge as is.  @agrare Thoughts here?

Reply:
Actually, will this work?  I thought response came _after_ we run the command (at least in sync mode)

Reply:
If you wanted to add something to the list of env_vars then up on line 215 would be the place

Reply:
I'm a little confused though, don't we already have this for vault credentials? https://github.com/ManageIQ/manageiq/blob/master/lib/ansible/runner/credential/vault_credential.rb#L9-L10

Reply:
@jaisejose1123 can you post what the env_vars_hash, extra_vars_hash, and params are by line 223 in your live test?  I'm curious if the ANSIBLE_VAULT_PASSWORD_FILE env var is missing somehow

Reply:
> I'm a little confused though, don't we already have this for vault credentials? https://github.com/ManageIQ/manageiq/blob/master/lib/ansible/runner/credential/vault_credential.rb#L9-L10

Yes - it almost feels like this test should work _without_ the code change...let me try that locally.

Reply:
@jaisejose1123 Yeah, as expected, if I remove all of the code changes here in lib/ansible/runner.rb, and then run the new spec below, it still passes.  Do you have a spec that shows the failure before a code change?

Reply:
At first, it wasn't working from the front end with the existing code. After I made those changes, it started working. However, the encrypted playbook is still running with the old code,